packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator.ClientModel/src/Providers/Abstractions/HttpRequestApi.cs

```diff
@@ -24,5 +24,6 @@ protected HttpRequestApi(CSharpType type, ValueExpression original) : base(type,
         public abstract ValueExpression Content();
         public abstract HttpRequestApi FromExpression(ValueExpression original);
         public abstract HttpRequestApi ToExpression();
+        public abstract string? ClientRequestIdPropertyName { get; }
```

Comment:
why we must put a `XXXName` here?
These abstractions should be returning the expressions such as
```
public abstract ScopedApi<string>? ClientRequestId();
```
like we have above for `Content` - we did not have a `HttpRequestContentPropertyName`, instead we made a thing to return the corresponding expression.